content/en/docs/components/training/overview.md

```diff
@@ -1,65 +1,65 @@
+description = "An overview of the Training Operator"
 weight = 10
 +++
 
 {{% stable-status %}}
 
-## What is Training Operator ?
+## What is the Training Operator?
 
-Training Operator is a Kubernetes-native project for fine-tuning and scalable
-distributed training of machine learning (ML) models created with various ML frameworks such as
+The Training Operator is a Kubernetes-native project for fine-tuning and scalable
+distributed training of machine learning (ML) models created with different ML frameworks such as
 PyTorch, TensorFlow, XGBoost, and others.
 
-User can integrate other ML libraries such as [HuggingFace](https://huggingface.co),
+You can integrate other ML libraries such as [HuggingFace](https://huggingface.co),
 [DeepSpeed](https://github.com/microsoft/DeepSpeed), or [Megatron-LM](https://github.com/NVIDIA/Megatron-LM)
-with Training Operator to orchestrate their ML training on Kubernetes.
+with the Training Operator to orchestrate their ML training on Kubernetes.
 
-Training Operator allows you to use Kubernetes workloads to effectively train your large models
-via Kubernetes Custom Resources APIs or using Training Operator Python SDK.
+The Training Operator allows you to use Kubernetes workloads to effectively train your large models
+via Kubernetes Custom Resources APIs or using the Training Operator Python SDK.
 
-Training Operator implements centralized Kubernetes controller to orchestrate distributed training jobs.
+The Training Operator implements a centralized Kubernetes controller to orchestrate distributed training jobs.
 
-Users can run High-performance computing (HPC) tasks with Training Operator and MPIJob since it
+You can run High-performance computing (HPC) tasks with the Training Operator and MPIJob since it
 supports running Message Passing Interface (MPI) on Kubernetes which is heavily used for HPC.
-Training Operator implements V1 API version of MPI Operator. For MPI Operator V2 version,
+The Training Operator implements the V1 API version of MPI Operator. For the MPI Operator V2 version,
 please follow [this guide](/docs/components/training/user-guides/mpi/) to install MPI Operator V2.
 
 <img src="/docs/components/training/images/training-operator-overview.drawio.png"
   alt="Training Operator Overview"
   class="mt-3 mb-3">
 
-Training Operator is responsible for scheduling the appropriate Kubernetes workloads to implement
+The Training Operator is responsible for scheduling the appropriate Kubernetes workloads to implement
 various distributed training strategies for different ML frameworks.
 
-## Why Training Operator ?
+## Why use the Training Operator?
 
-Training Operator addresses Model Training and Model Fine-Tuning step in AI/ML lifecycle as shown on
-that diagram:
+The Training Operator addresses the Model Training and Model Fine-Tuning steps in the AI/ML
+lifecycle as shown in diagram below:
 
 <img src="/docs/components/training/images/ml-lifecycle-training-operator.drawio.svg"
```

Comment:
It would be amazing, to update this diagram with the latest changes (e.g. different colour for boxes + model registry).
E.g. @tarilabs updated it for Model Registry: https://www.kubeflow.org/docs/components/model-registry/overview/#what-is-model-registry

Reply:
Indeed the main architecture drawio diagram proved re-usable, thanks again @andreyvelich

Reply:
Sounds good. I noticed that one is missing the feature extraction components that are listed in the architecture page here: https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle

Should I just update them all to point to that one?

Reply:
I think, we should just show how Training Operator fits into ML Lifecycle. E.g. we don't need to show each Kubeflow component in that diagram, only Training Operator.
Similar to what we have right now: https://www.kubeflow.org/docs/components/training/overview/#why-training-operator-
We just need to update this diagram.

Reply:
Are you looking for a diagram similar to the model registry:

![Screenshot 2024-08-09 at 11 20 09 AM](https://github.com/user-attachments/assets/d6b4c2ba-940c-49f8-ab0f-cc150e024828)

Do you want it to highlight "inner loop" and "outer loop" as well? Or are you looking to diagram something else? 

From my perspective, Training/Fine-Tuning produces the core artifact, so, in many ways, it's the "brain" of the whole operation, which means everything else is connected to it. 

Would be helpful to know a little bit more about what you're thinking.

Reply:
No, I am just asking to update [this diagram](https://www.kubeflow.org/docs/components/training/overview/#why-training-operator-) with [the updated version](https://www.kubeflow.org/docs/started/architecture/#introducing-the-ml-lifecycle).
E.g. Use green text boxes, add model registry, add feature extraction and generation boxes.

Reply:
Oh I see the difference now, my bad. Took a bit. I updated. LMK if that covered it.

Reply:
Sounds great! Maybe similar to Model Registry, we can make the text bigger and highlight it with blue colour.